### PR TITLE
Updating readme and gitignore and fixing a couple error/warnigns

### PR DESCRIPTION
**Pull request by plauche**

### CODE REVIEW

File: server/package-lock.json [hunk 1]
```diff
@@ -1,609 +0,0 @@
-{
```
Comment (plauche) [inline server/package-lock.json:1]: Yarn was complaining about this file existing

File: README.md [hunk 2]
```diff
@@ -2,8 +2,22 @@ Arcas load testing depends on GStreamer version 1.18.X being installed on your c
...
+  mkdir -p /usr/local/opt/gst-plugins-base
+  ln -s /Library/Frameworks/GStreamer.framework/Versions/1.0/lib /usr/local/opt/gst-plugins-base/lib
+  mkdir -p /usr/local/opt/gstreamer
+  ln -s /Library/Frameworks/GStreamer.framework/Versions/1.0/lib /usr/local/opt/gstreamer/lib
+  ```
+
+- Windows will need to install both of the following:
+  - https://gstreamer.freedesktop.org/data/pkg/windows/1.18.6/msvc/gstreamer-1.0-msvc-x86-1.18.6.msi
+  - https://gstreamer.freedesktop.org/data/pkg/windows/1.18.6/msvc/gstreamer-1.0-devel-msvc-x86-1.18.6.msi
+
```
Comment (aidylka) [inline README.md]: Add this after line 151:

```
Note: If this produces an error loading gst-plugins-base, make sure your GStreamer version is 1.18.X and that you followed the installation instructions above.
```
and maybe add a link here

File: README.md [hunk 3]
```diff
@@ -2,8 +2,18 @@ Arcas load testing depends on GStreamer version 1.18.X being installed on your c
...
+  
+  After installing both of those packages, please run the following commands in the terminal:
+  ```
+  mkdir -p /usr/local/opt/gst-plugins-base
+  ln -s /Library/Frameworks/GStreamer.framework/Versions/1.0/lib /usr/local/opt/gst-plugins-base/lib
+  mkdir -p /usr/local/opt/gstreamer
+  ln -s /Library/Frameworks/GStreamer.framework/Versions/1.0/lib /usr/local/opt/gstreamer/lib
+  ```
+
 - Linux - Please see the GStreamer installation [guide](https://gstreamer.freedesktop.org/documentation/installing/on-linux.html?gi-language=c)
```
Comment (aidylka) [inline README.md]: Let's move the Linux scenario first so that there isn't that much space between the 2 bullets

**Review by aidylka — APPROVED** [review]
LGTM!

File: README.md [hunk 4]
```diff
@@ -135,6 +149,8 @@ In a separate terminal window in the `/arcas-load-test-mediasoup-example` direct
 yarn && yarn server
 ```
 
+> Note: If this produces an error loading gst-plugins-base, make sure your Gstreamer version is 1.18.x and that you have followed the installation instructions above.
```
Comment (aidylka) [inline README.md]: Typo on `GStreamer`

**Pull request merged**